src/lib/models/outoforder/Core.cc

```diff
@@ -223,16 +224,12 @@ std::map<std::string, std::string> Core::getStats() const {
   auto backendStalls = dispatchIssueUnit_.getBackendStalls();
   auto portBusyStalls = dispatchIssueUnit_.getPortBusyStalls();
 
-  uint64_t totalBranchesExecuted = 0;
-  uint64_t totalBranchMispredicts = 0;
+  uint64_t totalBranchesFetched = fetchUnit_.getBranchFetchedCount();
+  uint64_t totalBranchesRetired = reorderBuffer_.getRetiredBranchesCount();
+  uint64_t totalBranchMispredicts = reorderBuffer_.getBranchMispredictedCount();
 
-  // Sum up the branch stats reported across the execution units.
-  for (auto& eu : executionUnits_) {
-    totalBranchesExecuted += eu.getBranchExecutedCount();
-    totalBranchMispredicts += eu.getBranchMispredictedCount();
-  }
-  auto branchMissRate = 100.0f * static_cast<float>(totalBranchMispredicts) /
-                        static_cast<float>(totalBranchesExecuted);
+  auto branchMissRate = 100.0f * static_cast<double>(totalBranchMispredicts) /
```

Comment:
A bit pedantic.... but this is multiplying a float and a double, so should be `100.0 * ...`

Reply:
I live for the pedantry